src/highdicom/sr/templates.py

```diff
@@ -3479,33 +3645,32 @@ def from_sequence(
 
 class ImageLibraryEntryDescriptors(Template):
 
-    """`TID 1602 <http://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_A.html#sect_TID_1602>`_
-     Image Library Entry Descriptors"""  # noqa: E501
+    """:dcm:"`TID 1602 <part16/chapter_A.html#sect_TID_1602>`_
+     Image Library Entry Descriptors"""
 
     def __init__(
         self,
-        modality: Union[Code, CodedConcept],
-        frame_of_reference_uid: str,
-        pixel_data_rows: int,
-        pixel_data_columns: int,
+        image: Dataset,
         additional_descriptors: Optional[Sequence[ContentItem]] = None
     ) -> None:
         """
+
         Parameters
         ----------
-        modality: Union[highdicom.sr.CodedConcept, pydicom.sr.coding.Code]
-            Modality
-        frame_of_reference_uid: str
-            Frame of Reference UID
-        pixel_data_rows: int
-            Number of rows in pixel data frames
-        pixel_data_columns: int
-            Number of rows in pixel data frames
-        additional_descriptors: Union[Sequence[highdicom.sr.ContentItem], None], optional
-            Additional SR Content Items that should be included
+        image: pydicom.Dataset
+            Metadata of a referenced image instance
+        additional_descriptors: Sequence[highdicom.sr.ContentItem],
```

Comment:
This is how we have been doing parameter types elsewhere
```suggestion
        additional_descriptors: Union[Sequence[highdicom.sr.ContentItem], None], optional
```